assets/prototype/application/utilities/money/test/formatAmount.test.ts

```diff
@@ -0,0 +1,32 @@
+import { formatAmount } from '../';
+
+describe('formatAmount', () => {
+	it('checks for formatted amount to be a string', () => {
+		const decimalPlaces = 2;
+
+		const format = formatAmount(decimalPlaces);
+		const result = format(5);
+
+		expect(typeof result).toBe('string');
+	});
+
+	it('checks for formatted amount for positive number of deicmal places', () => {
+		const decimalPlaces = 2;
+
+		const format = formatAmount(decimalPlaces);
+
+		expect(format(5)).toEqual('5.00');
+		expect(format(5.2)).toEqual('5.20');
+		expect(format(-0.5)).toEqual('-0.50');
+	});
+
+	it('checks for formatted amount for 0 deicmal places', () => {
+		const decimalPlaces = 0;
+
+		const format = formatAmount(decimalPlaces);
+
+		expect(format(5.0)).toEqual('5');
+		expect(format(5.2)).toEqual('5');
+		expect(format(-0.5)).toEqual('-1');
```

Comment:
should we test for
`expect(format(5.5)).toEqual(...);`
or
`expect(format(5.49)).toEqual(...);`
?

Reply:
as I wrote above, for low-level utilities, I would prefer to see them getting hammered with as many tests as possible to confirm their behaviour under any and all possible scenarios that could potentially occur at runtime. I also think loops make it easier to write tests for these kinds of functions since the test cases area so straightforward and consistent to test